src/core/stream.h

```diff
@@ -363,10 +363,20 @@ typedef struct QUIC_STREAM {
     //
     uint16_t SendPriority;
 
+    //
+    // If > 0, bytes up to offset must be re-transmitted and ACK'd from peer before we can abort this stream.
+    //
+    uint64_t ReliableOffsetSend;
```

Comment:
For padding (extra space in the struct) reasons you want to put this `uint64_t` after one of the other `uint64_t` above.

Reply:
Can update.